.github/workflows/cicd.yml

```diff
@@ -12,55 +12,45 @@ jobs:
   dartdoc:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v1
-      - uses: actions/setup-java@v1
-        with:
-          java-version: '12.x'
+      - uses: actions/checkout@v2
       - uses: subosito/flutter-action@v1
         with:
-          channel: 'beta'
-      - run: flutter pub get # make sure flutter downloads dart for us
-      - run: ./scripts/dartdoc.sh
+          channel: 'stable'
+      - uses: flame-engine/flame-dartdoc-action@v2
+        with:
+          package-path: './pubspec.yaml'
 
   format:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v1
-      - uses: actions/setup-java@v1
-        with:
-          java-version: '12.x'
+      - uses: actions/checkout@v2
       - uses: subosito/flutter-action@v1
         with:
-          channel: 'beta'
-      - run: flutter pub get # make sure flutter downloads dart for us
-      - run: ./scripts/format.sh
+          channel: 'stable'
+      - uses: flame-engine/flame-format-action@v1
 
   analyze:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v1
-      - uses: actions/setup-java@v1
-        with:
-          java-version: '12.x'
+      - uses: actions/checkout@v2
       - uses: subosito/flutter-action@v1
         with:
-          channel: 'beta'
-      - run: flutter pub get # make sure flutter downloads dart for us
-      - run: ./scripts/analyze.sh
+          channel: 'stable'
+      - uses: flame-engine/flame-analyze-action@v2
+      - uses: coverallsapp/github-action@master
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
why does the analyze action needs our token?

Reply:
It's the coveralls app that needs it to be able to post on the PR I think